### PR TITLE
Rubicon Bid Adapter: Fix non IE11 Test .remove()

### PR DESCRIPTION
**Pull request by robertrmartinez**
## Type of change
- [X] Bugfix

## Description of change
IE does not support element.remove()

So we use parentNode.removeChild

### CODE REVIEW

**Review by patmmccann — APPROVED** [review]

**Comment by patmmccann** [comment]
this mirrors the logic in the polyfill ty

**Comment by robertrmartinez** [comment]
Let's make sure this passes tests before we merge @ChrisHuie @harpere

**Comment by patmmccann** [comment]
you want to wait for lgtm?

**Comment by robertrmartinez** [comment]
No good for me now. 👍

**Pull request merged**
